src/components/client/notifications/OverlayModal.tsx

```
@@ -0,0 +1,40 @@
+            onClick={onClose}
+            className="w-fit sticky top-0 text-gray-500 hover:text-gray-700 hover:cursor-pointer ml-auto"
+          >
+            x
+          </button>
+          {title && <h2 className="text-lg font-bold mb-4">{title}</h2>}
+          <div className="flex-1 flex flex-col">{children}</div>
+        </div>
+      </div>
+    );
+  }
+);
```

ESLint

[error] Component definition is missing display name

(react/display-name)